meson_options.txt

```diff
@@ -15,6 +15,7 @@ option('rest', type : 'feature', value : 'enabled', description : '''Enable Phos
 option('event-subscription', type : 'feature', value : 'enabled', description: 'Enable Event Subscription through websocket')
 option('redfish', type : 'feature',value : 'enabled', description: 'Enable Redfish APIs.  Paths are under \'/redfish/v1/\'. See https://github.com/openbmc/bmcweb/blob/master/DEVELOPING.md#redfish.')
 option('host-serial-socket', type : 'feature', value : 'enabled', description : 'Enable host serial console WebSocket. Path is \'/console0\'.  See https://github.com/openbmc/docs/blob/master/console.md.')
+option('hypervisor-serial-socket', type : 'feature', value : 'enabled', description : 'Enable hypervisor serial console WebSocket. Path is \'/console1\'.')
```

Comment:
Should this be enabled by default for all platforms?

Reply:
Upstream should be disabled, downstream since we only build for tacoma and rainier this is probably okay but it would be better if we disable this and only enable for those systems